Reduce footprint of wiring classes generated by Gizmo 2

gsmet opened the pull request:
In all these wiring classes, we can do without parameters or debug info.

Gizmo 1 weren't adding parameters and debug info so we are on par with what we used to do with Gizmo 1.

Ladicek reviewed (approved):
LGTM.

For the record, we could easily avoid the whitespace changes by not replacing

```java
Gizmo.create(classOutput).class_(generatedClassName, cc -> {
```

with

```java
Gizmo.create(classOutput)
        .withDebugInfo(false)
        .withParameters(false)
        .class_(generatedClassName, cc -> {
```

but instead with

```java
Gizmo gizmo = Gizmo.create(classOutput)
        .withDebugInfo(false)
        .withParameters(false);

gizmo.class_(generatedClassName, cc -> {

gsmet committed:
Reduce footprint of wiring classes generated by Gizmo 2

In all these wiring classes, we can do without parameters or debug info.

Gizmo 1 weren't adding parameters and debug info so we are on par with
what we used to do with Gizmo 1.

Ladicek reviewed (approved):
Perfect, thanks!
